.github/workflows/tests.yml

```diff
@@ -0,0 +1,19 @@
+name: Test the code
```

Comment:
Are we "testing" the code here or "formatting" the code or something else? The word "test" is usually used to ensure that the code gives correct results. Here, to my understanding, we are doing more of a syntax check etc. Right? What would be a better term? Evaluate code?

Reply:
@gregorgorjanc We are evaluating the format of the code now. The reason I used the "tests" here  is becasue I expect someday after we might add real code tests into this workflow, so the "checking the format of the code" plus "checking the outcomes of the code" becomes the complete tests workflow.

However, I can change the name of the workflow for now and rename it later when the real tests really applied.

Reply:
@XingerTang ok leave the file name but please change the name to "Evaluate code syntax (and correctness in the future)". Thanks!

Reply:
@gregorgorjanc Done!